src/server/protocol.ts

```diff
@@ -2633,6 +2633,21 @@ namespace ts.server.protocol {
         insertSpaceBeforeTypeAnnotation?: boolean;
     }
 
+    export interface Options {

```

Comment:
`UserPreferences`? Thoughts @RyanCavanaugh? (keep https://github.com/Microsoft/TypeScript/pull/22236#discussion_r171675948 in mind)